src/routes/productRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const productController = require('../controllers/productController');
+const errorHandler = require("../middlewares/errorHandler");
+
+const router = express.Router();
+
+router.get("/main", errorHandler(productController.productAllList));
+router.get("/main/aroma/:aromasId", errorHandler(productController.productAromaList));
+router.get("/main/:categoryId", errorHandler(productController.productCategoryList));
+router.get("/main/detail/:productId", errorHandler(productController.productDetailPage));
```

Comment:
엔드포인트가 페이지 기준으로 만들어져 있습니다.
프론트엔드 페이지가 아니라 자원(데이터) 기준으로 만들어주셔야 Restful하게 엔드포인트를 설계할 수 있습니다.

```suggestion
router.get("", errorHandler(productController.productAllList));
router.get(":productId", errorHandler(productController.productDetailPage));
```